src/style/themes/base/base-theme.config.js

```diff
@@ -13,6 +13,7 @@ export default (palette) => {
       secondary: palette.genericGreenShade(35),
       tertiary: palette.genericGreenShade(55),
       brand: palette.brilliantGreen,
+      whiteMix: '#e5f2ee',
```

Comment:
is this color hover color for all themes?

Reply:
I'm not sure about the colors in themes that they are not created by palette. @samhinton88 Could you take a look at this? I had similar problem before but I don't remember exactly what was that about